packages/probers/cypress/e2e/signUp.cy.ts

```diff
@@ -63,12 +63,15 @@ describe('Sign Up', () => {
 
     it('can navigate to sign-up from the public site', () => {
       cy.visit('')
-      cy.findByRole('button', { name: /sign up free/i }).click()
-
+      cy.findByTestId('signUpButtonDesktop')
```

Comment:
what's up with this? im thinking we refactor the code to be a link so we can simply do `findByRole('link', {name: /sign up/i}).click()

Reply:
Yeah curious if you have any ideas here. The issue is that the mobile nav bar still renders even when not on mobile, so there are two of these buttons on the page with the same role/name

Reply:
Ah gotcha, in that care you can do some "within" code, so you can be like within main ...

Reply:
🙈 after pondering this I realized the obvious solution is just adding `aria-hidden` to the overlay when it's not showing

Reply:
yeah for sure, nice :)